schema.json

```diff
@@ -1,255 +1,266 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter Metadata Language Schema, Informative",
+  "description": "The (informative) schema for the Parameter Metadata Language
```

Comment:
I'm not sure this is valid JSON based on the discussion here, we might need to tweak our linter: https://stackoverflow.com/questions/2392766/multiline-strings-in-json

Reply:
Indeed this validator doesn't like the whitespace:
https://jsonlint.com/

Reply:
Fixed in https://github.com/OpenLightingProject/pid-schema/pull/11/commits/132c5adf3e59b8718dce77f7965d18ff72deef35 in #11 .